app/views/layouts/_header.html.erb

```diff
@@ -49,6 +49,12 @@
             <% end %>
             <li>
               <i class="fa-solid fa-question"></i>
+              <a href="https://github.com/MarkUsProject/Wiki/blob/release/Home.md">
```

Comment:
oh, you should use the `markus_version` config value (see other places that link to the wiki)